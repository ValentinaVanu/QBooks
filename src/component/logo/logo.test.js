import { Logo } from '.'
import renderer from 'react-test-renderer'


test('renders main logo component', () => {
  const logo = renderer.create(<Logo />).toJSON()
  expect(logo).toMatchSnapshot()
})
