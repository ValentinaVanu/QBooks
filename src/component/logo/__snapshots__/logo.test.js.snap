// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders main logo component 1`] = `
<a
  aria-current="page"
  className="sc-bdfBwQ ffIExw"
  href="/"
  onClick={[Function]}
/>
`;
